### ISSUE TITLE
badge doesn't increment

### ISSUE THREAD

**Issue by mounabenayed** [Wolox/react-chat-widget#121]
I used the react-chat-widget to create a bot.
When the bot replies to the user I want to increments the value of the badge. I created a global var that increments when the bot answers and back to zero when the user write his input. the I pass it to the badge as the following 
`
  render() {
    return (<Widget
      badge={$badgevalue}
      handleNewUserMessage={this.handleNewUserMessage}
      profileAvatar={profileAvatar}
      subtitle=""
      title="Chatbot"
    />
    );
 `
but it seems not be working. 
So I am wondering how it does work and how can I resolve my issue

**Comment by ashadnasim52** [comment]
You need to pass number in badge
for example

`badge='4'`

here your working code

`render() { return (<Widget badge=4 handleNewUserMessage={this.handleNewUserMessage} profileAvatar={profileAvatar} subtitle="" title="Chatbot" /> );`

**Comment by snetts** [comment]
You need to count the messages on the addResponseMessage that updates your badge. Maybe push the messages into an array and get the length of the array and use that to update the badge.

**Comment by kartikmaan** [comment]
Can someone suggest a more clear approach towards this

**Comment by Bonfims** [comment]
the standart `addResponseMessage` method set a number to badge count, when widget is in minimized state.

but, when use the `renderCustomComponent` method does not set a number for the badge count.

![image](https://user-images.githubusercontent.com/16294667/93773783-df979000-fbf6-11ea-8d52-03d062666c75.png)

**Comment by jawetzel** [comment]
I agree with @kartikmaan. This could use a more clear way to be done. Either exposing the current count so we can pass a new value to increment it, or a method to increment the value by a given number. As it stands now the only accurate way I could come up with:

1. Set bool stating away on minimize or instantiated without opening.
2. Set a counter that keeps track of event counts until app is opened.
3. On open set bool to false and clear count
